package.json

```diff
@@ -25,6 +25,7 @@
     "node-cron": "^2.0.1",
     "prom-client": "^11.1.3",
     "request": "^2.88.0",
```

Comment:
It isn't necessary anymore, is it?

Reply:
It's necessary because request is defined as a peer-dependency.
https://github.com/request/request-promise#installation

Reply:
:+1: